Marlin/temperature.h

```diff
@@ -303,6 +303,37 @@ class Temperature {
       #endif
     }
 
+    #if ENABLED(BABYSTEPPING)
+
+      FORCE_INLINE void babystep_axis(AxisEnum axis, int distance) {
```

Comment:
It should. Not sure if it matters, but it's an _instance_ method, and  not a _class_ method, and since it's defined in the header, it should inline everywhere.